crypto-core/src/main/java/com/palantir/crypto/io/DecryptingSeekableInput.java

```diff
@@ -32,8 +32,12 @@
     private final SeekableCipher seekableCipher;
     private final CipherStreamSupplier supplier;
 
-    private CipherInputStream decryptedStream;
-    private long decryptedStreamPos;
+    // block info
+    private long blockIndex = -1;
```

Comment:
should add some comment here to clarify what exactly these are, invariants,e tc.